lib/CppInterOp/CppInterOp.cpp

```diff
@@ -716,6 +716,18 @@ TCppScope_t GetParentScope(TCppScope_t scope) {
 TCppIndex_t GetNumBases(TCppScope_t klass) {
   auto* D = (Decl*)klass;
 
+  if (auto* CTSD = llvm::dyn_cast_or_null<ClassTemplateSpecializationDecl>(D)) {
+    if (!CTSD->hasDefinition())
+#if CLANG_VERSION_MAJOR < 20
+      getSema().InstantiateClassTemplateSpecialization(
+          SourceLocation::getFromRawEncoding(1), CTSD,
+          clang::TemplateSpecializationKind::TSK_Undeclared, true);
+#else
+      getSema().InstantiateClassTemplateSpecialization(
+          SourceLocation::getFromRawEncoding(1), CTSD,
+          clang::TemplateSpecializationKind::TSK_Undeclared, true, false);
+#endif
```

Comment:
That block is a good candidate for our `lib/CppInterOp/Compatibility.h` layer. Please also spell out the names of the default args.